HHS/Head-Start-TTADP
# Styling fixes for the Grantee TTA Record page

## Description of change
- Move export reports button closer to activity reports title
- Spent a little bit of time trying to fix some issues on smaller screens. Between those issues and the previous one, I wanted to refactor the table headers/menu/pagination to be a bit more naturally flexible. Created a new component to prevent a bunch of duplicated markup.
- Set width for grantee tta record page
- Bottom align headings in target pops table
- Fixed legends on regional dashboard

- ~Make all Target Populations widget look like a table~ (fixed on https://github.com/HHS/Head-Start-TTADP/pull/624)
- ~Use the overridden table stripe color~ (fixed on https://github.com/HHS/Head-Start-TTADP/pull/624)

## How to test
Visit the affected areas on the site and confirm the UI is changed (and still works)

## Issue(s)
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-466

## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
